tests/Unit/Domain/Creators/Test_CylindricalBinaryCompactObject.cpp

```diff
@@ -364,8 +368,11 @@ void test_connectivity() {
         CAPTURE(include_inner_sphere_A);
         for (const bool include_inner_sphere_B : {true, false}) {
           CAPTURE(include_inner_sphere_B);
-          test_connectivity_once(with_sphere_e, include_inner_sphere_A,
-                                 include_inner_sphere_B, include_outer_sphere);
+          for (const bool use_equiangular_map : {true, false}) {
```

Comment:
Use the new `cartesian_product` you avoid the nested loops. Also CAPTURE the bool.

Reply:
I capture the bool now.

I don't see how to use `cartesian_product` except for doing things that would make the code unreadable and/or bug prone, like creating a `cartesian_product({{false}}, {{true, false}}, {{true, false}}, {{true, false}},{{true, false}})` and then looping over it and trying to name the different fields `include_inner_sphere_A` etc (and get the order right) and CAPTURE them.
Maybe `cartesian_product` needs better documentation or an example?